docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -109,24 +109,9 @@ Drop the ``--user`` option in the following commands if you use a virtual enviro
 
    .. code-block:: bash
 
-    pip install --user -r tests/requirements.in -c tests/requirements.txt # Installs tested dependency versions.
-    pip install --user -r tests/requirements.in # Installs the unpinned dependency versions.
-    pip install --user -r tests/requirements-relaxed.in # Installs the unpinned dependency versions including untested antsibull-docs.
-
-.. note::
-
-    You may need to install these general pre-requisites separately on some systems:
-    - ``gcc``
-    - ``libyaml``
-    - ``make``
-    - ``pyparsing``
-    - ``wheel``
-    - ``six``
-    On macOS with Xcode, you may need to install ``six`` and ``pyparsing`` with ``--ignore-installed`` to get versions that work with ``sphinx``.
-
-.. note::
-
-  	After checking out ``ansible/ansible-documentation``, make sure the ``docs/docsite/rst`` directory has strict enough permissions. It should only be writable by the owner's account. If your default ``umask`` is not 022, you can use ``chmod go-w docs/docsite/rst`` to set the permissions correctly in your new branch.  Optionally, you can set your ``umask`` to 022 to make all newly created files on your system (including those created by ``git clone``) have the correct permissions.
```

Comment:
Hmm... I'm all for trimming out content that is no longer needed but I wonder if this is somehow losing a bit of history and worry it might confuse some users. What if we rephrased this to make it stand out as deprecated? That way we could perhaps give it a release cycle or so before completely dropping it.

Reply:
Are you referring to the part about manually installing Python packages (we use venv+pip for that now) and manually installing devel tools (everything we need should have pre-built wheels on PyPI) or the part about setting permissions?

Reply:
Both. Maybe users don't want to use a venv - or weren't using a venv previously - and that list of packages might still be relevant.

I guess with the move from `ansible/ansible` the part about permissions is no longer relevant. Maybe it's likely no users will ever hit this again and I'm being overly cautious about dropping it. And it might not make that much sense to point out something you used to need to do but now don't.

Reply:
Even for developers not using venvs[^1], the information about prerequisites is entirely out of date. `gcc`, `libyaml`, and `make` shouldn't be necessary due to the aforementioned pre-compiled wheels. Newer versions of pip build wheels in an isolated environment and don't need `wheel` installed manually. `six` and `pyparsing` shouldn't need to be installed manually, and the latter is no longer part of our dependency tree.

[^1]: which is a bad idea. Installing a lockfile into a global environment is quite likely to break things